### PR TITLE
boards : renamed opencm9-04 into opencm904

### PR DESCRIPTION
**Pull request by astralien3000**
The title speaks for itself.

### CODE REVIEW

File: boards/opencm904/include/board.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    boards_opencm9-04 OpenCM9.04
+ * @defgroup    boards_opencm904 OpenCM9.04
```
Comment (aabadie) [inline boards/opencm904/include/board.h]: Maybe also rename OpenCM9.04 ?

Comment (astralien3000) [inline boards/opencm904/include/board.h]: Why ?

Comment (astralien3000) [inline boards/opencm904/include/board.h]: OpenCM9.04 is the "real" name of the board, why changing it ?

**Comment by astralien3000** [comment]
I just squashed the commits. I hope it is OK.

File: boards/opencm904/include/board.h (hunk 1 above)
Comment (aabadie) [inline boards/opencm904/include/board.h]: ok, sorry, you are right.

**Comment by astralien3000** [comment]
Oops ! 2 commits were missing, sorry.

**Comment by aabadie** [comment]
indeed, that's why the CI exists :)

**Comment by aabadie** [comment]
needs rebase as #6589 has just been merged.

**Comment by astralien3000** [comment]
rebased

**Comment by aabadie** [comment]
needs rebase again

**Comment by astralien3000** [comment]
Done ! And CI passed !

**Comment by aabadie** [comment]
then go !

**Pull request merged**